docs/modules/usage/how-to/gui-mode.md

```diff
@@ -19,6 +19,15 @@ OpenHands provides a user-friendly Graphical User Interface (GUI) mode for inter
 3. Enter the corresponding `API Key` for your chosen provider.
 4. Click "Save" to apply the settings.
 
+### GitHub Token Setup
+
+You can provide a `GITHUB_TOKEN` is provided to OpenHands through two methods:
+
+1. When running locally: You can manually input your GitHub token
+2. When using online: The token is obtained through GitHub OAuth authentication
+
+If this token exists, it is automatically exported to the shell environment for the agent to use.
```

Comment:
```suggestion
OpenHands automatically exports a `GITHUB_TOKEN` to the shell environment if it is available. This can happen in two ways:

1. Locally (OSS): The user directly inputs their GitHub token.
2. Online (SaaS): The token is obtained through GitHub OAuth authentication.

When you reach the `/app` route, the app checks if a token is present. If it finds one, it sets it in the environment for the agent to use.
```

Reply:
IMO the original draft makes it sound like it is up to the user to set the GH token, this change makes it clearer that it is set automatically given the conditions